flink-libraries/flink-table/src/test/scala/org/apache/flink/table/expressions/utils/ScalarTypesTestBase.scala

```diff
@@ -65,6 +65,9 @@ class ScalarTypesTestBase extends ExpressionTestBase {
     testData.setField(33, null)
     testData.setField(34, 256)
     testData.setField(35, "aGVsbG8gd29ybGQ=")
+    testData.setField(36, 65)
+    testData.setField(37, 97)
```

Comment:
`f36` and `f37`, I think we can only keep one of them.

Reply:
agree.